reference/docs-conceptual/discover-powershell.md

```diff
@@ -1,26 +1,23 @@
 ---
 description: Learn what PowerShell is and some essential commands used to discover more about PowerShell.
-ms.custom: chnoring
-ms.date: 07/12/2022
+ms.date: 01/31/2023
 ms.reviewer: sewhee
 title: Discover PowerShell
 ---
 
 # Discover PowerShell
 
 PowerShell is a command-line shell and a scripting language in one. PowerShell started out on
-Windows. It was meant to help with task automation for administration tasks but has now grown to be
-cross platform and can be used for a variety of tasks.
+Windows to help automate administrative tasks. Now, it runs cross platform and can be used for
+various tasks.
 
-The thing that makes PowerShell unique is that it accepts and returns .NET objects, rather than text.
-This fact makes it easier to connect different commands in series, in a _pipeline_.
-
-Even then, you might need to _massage_ the results a little.
+The thing that makes PowerShell unique is that it accepts and returns .NET objects, rather than
+text. This feature makes it easier to connect different commands in a _pipeline_.
 
 ## What can PowerShell be used for?
 
 Usage of PowerShell has grown since the days when it was Windows-only. It's still used for Windows
-task automation, but today, you can use for a variety of tasks like:
+task automation, but today, you can use for tasks like:
```

Comment:
My brain thinks this is a missing word, but it wasn't in the original either, so I'm not extremely confident I'm correct here.

```suggestion
task automation, but today, you can use it for tasks like:
```